apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.html.ts

```diff
@@ -38,89 +40,95 @@ export function InstructorInstanceAdminSettings({
-              forward slashes to separate directories. The recommended format is
-              <code>Fa19</code> or <code>Fall2019</code>. Add suffixes if there are multiple
-              versions, like <code>Fa19honors</code>.
-            </small>
-          </div>
-          <div class="form-group">
-            <label for="student_link">Student Link</label>
-            <span class="input-group">
-              <span readonly class="form-control form-control-sm">${studentLink}</span>
-              <div class="input-group-append">
-                <button
-                  type="button"
-                  class="btn btn-sm btn-outline-secondary btn-copy"
-                  data-clipboard-text="${studentLink}"
-                  aria-label="Copy student link"
-                >
-                  <i class="far fa-clipboard"></i>
-                </button>
-                <button
-                  type="button"
-                  title="Student Link QR Code"
-                  aria-label="Student Link QR Code"
-                  class="btn btn-sm btn-outline-secondary js-qrcode-button"
-                  data-qr-code-content="${studentLink}"
-                >
-                  <i class="fas fa-qrcode"></i>
-                </button>
-              </div>
-            </span>
-            <small class="form-text text-muted">
-              This is the link that students will use to access the course. You can copy this link
-              to share with students.
-            </small>
-          </div>
-          ${EditConfiguration({
-            hasCoursePermissionView: resLocals.authz_data.has_course_permission_view,
-            hasCoursePermissionEdit: resLocals.authz_data.has_course_permission_edit,
-            exampleCourse: resLocals.course.example_course,
-            urlPrefix: resLocals.urlPrefix,
-            navPage: resLocals.navPage,
-            infoCourseInstancePath,
-          })}
+          <form name="edit-course-instance-settings-form" method="POST">
+            <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+            <input type="hidden" name="orig_hash" value="${origHash}" />
+            <div class="form-group">
+              <label for="long_name">Long Name</label>
+              <input
+                type="text"
+                class="form-control"
+                id="long_name"
+                name="long_name"
+                value="${resLocals.course_instance.long_name}"
+                required
+                ${resLocals.authz_data.has_course_permission_edit &&
+                !resLocals.course.example_course
+                  ? ''
+                  : 'disabled'}
+              />
+              <small class="form-text text-muted">
+                The long name of this course instance (e.g., 'Spring 2015').
+              </small>
+            </div>
+            <div class="form-group">
+              <label for="ciid"> CIID </label>
```

Comment:
Nit, extraneous spaces.

```suggestion
              <label for="ciid">CIID</label>
```